.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+SCRIPT_DIR=$(dirname "${0}")
+. ${SCRIPT_DIR}/_/husky.sh
+npx lint-staged
```

Comment:
Can we now lose Husky as a dependency in the Editor if the pre-commit hook is running here?